### PR TITLE
Not my location

### PR DESCRIPTION
**Pull request by ItsKappaIsbae**
I found my **Location** and wanted to pin it @githubteacher 
closes#4030

### CODE REVIEW

**Comment by KarlaReyo** [comment]
I don´t really understand what's the problem but I hope you can solve it! 🎉

File: _pins/ItsKappaIsbae.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+githubHandlel: ItsKappaIsbae
+lattiude: 45.389183
```
Comment (hectorsector) [inline _pins/ItsKappaIsbae.json]: @ItsKappaIsbae just correct spelling on the word `latitude` and it should work 👍

**Comment by brianamarie** [comment]
Hi @ItsKappaIsbae! I'm going to go ahead and make the changes now.

**Comment by brianamarie** [comment]
@ItsKappaIsbae I'm going to merge now. Nice work!

**Pull request merged**